catalog-rest-service/src/main/resources/json/schema/entity/services/connections/database/bigQueryConnection.json

```diff
@@ -98,5 +93,5 @@
     }
   },
   "additionalProperties": false,
-  "required": ["credentials"]
+  "required": ["credentials", "projectId"]
```

Comment:
projectId is actually not required, only if we pass the credentials as files instead of values. However, it might be interesting to make it mandatory and pick it always from here. Thoughts?